enhancements/update/cvo-log-level-api.md

```diff
@@ -124,15 +124,56 @@ It is needed to address the fact that the hosted cluster API server will now
 provide access to the new CR that represents the CVO configuration. A 
 hosted cluster administrator could set the log level of the hosted CVO in the
 management cluster, thus affecting the storage space of the management cluster.
+The `ClusterVersionOperator` `cluster` resource in the hosted cluster will be 
+protected by a validating admission policy to ensure a hosted cluster
+administrator cannot modify it and thus affect the management cluster ([as is 
+currently being done for certain resources in HyperShift][kas/admissionpolicies]).
+
+The `ClusterVersionOperator` `cluster` resource will be modifiable from the 
+management cluster. To achieve that, the [`HostedCluster`][HostedClusterAPI] 
+API will be extended, and its values for the `ClusterVersionOperator` 
+configuration will be propagated to the hosted cluster.
+
+The existing [`ClusterConfiguration`][ClusterConfigurationAPI] API could be 
+considered to reference the `ClusterVersionOperator` API; however, its 
+purpose is oriented around the configuration API (`github.com/openshift/api/config/v1`), 
+meaning configuration for OCP components rather than just operators.
+As such, a new API is proposed to extend the [`HostedCluster`][HostedClusterAPI]
+API. A new `OperatorConfiguration` API where various APIs from the 
+`github.com/openshift/api/operator/v1` package can be referenced, including 
+the newly proposed `ClusterVersionOperator` API.
+
+Starting in the alpha group, the following changes are proposed for the
+[`api/hypershift/v1alpha1/hostedcluster_types.go`][api/hypershift/v1alpha1/hostedcluster_types.go] 
+file.
```

Comment:
If this is going into an existing hypershift API, you should add it to the latest version of the HCP API only

Reply:
Thanks, I'll check for the exact distinctions between the `HostedCluster` and `HostedControlPlane` CRDs. It's true that the new API should be in the `HostedControlPlane`. Let me check if it should also be considered a part of the `HostedCluster`.

Reply:
I more meant, I believe they are on v1beta1 for their types now, so you'd be adding the field to an older API version if you added it to v1alpha1

Reply:
Oh, right, thanks :facepalm: